### DISCUSSION TITLE
Windows 11 taskbar at side of the screen?

### DISCUSSION THREAD

**Discussion by 117649** [valinet/ExplorerPatcher#807] [Ideas]
Windows 11 taskbar at right or left of the screen.
Do we have this? Can we have it?

**Comment by SergoSH** [comment]
You can place your taskbar on any side of the screen. But i couldn't place taskbar on my second screen anywhere.

**Comment by Poopooracoocoo** [comment]
> [In response to another feature request for the W11 taskbar that he may have the same thoughts on] That hope would better be directed at Microsoft. It's hard doing things with someone else's stuff, especially when the hostility towards interacting with it as as high as Microsoft's. 

Source: https://github.com/valinet/ExplorerPatcher/discussions/404#discussioncomment-1917059

More info: https://github.com/valinet/ExplorerPatcher/wiki/All-features#primarysecondary-taskbar-location-on-screen which says this isn't a feature of EP.

**Reply by SergoSH** [reply]
Thanks a lot!!!
This helped: Although it may not be obvious, when using the Windows 10 taskbar, you don't have to actually use this. You can actually achieve much more by right clicking any taskbar and unticking "Lock the taskbar" or "Lock all taskbars". Then, simply drag any of the taskbars to dock it to whatever edge of the screen you'd like.
